lms/templates/problem_ajax.html

```diff
@@ -1 +1 @@
-<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}"></div>
+<div id="problem_${element_id}" class="problems-wrapper" data-problem-id="${id}" data-url="${ajax_url}" data-progress_status="${progress_status}" data-progress_detail="${progress_detail}" data-content="${content | h}"></div>
```

Comment:
Update on this -- I tried to do it inline, got some bugs, spent more time on them than I cared to, and am now giving up. This PR will still give the benefit of not having the extra HTTP requests, even if doesn't improve the JS rendering overhead. We can get to that later.